### ISSUE TITLE
how to train own vq-f4

### ISSUE THREAD

**Issue by lihao9056** [CompVis/latent-diffusion#274]
Great work.But I would like to know how to train own vq-f4.

**Comment by Nanicy** [comment]
me too

**Comment by QiuWYZju** [comment]
you can refer first_stage_config of  'ffhq-ldm-vq-4.yaml' and write your own config,such as :

model:
  base_learning_rate: 4.5e-06
  target: taming.models.vqgan.VQModel
  params:
    embed_dim: 3
    n_embed: 8192
    ddconfig:
      double_z: false
      z_channels: 3
      resolution: 256
      in_channels: 3
      out_ch: 3
      ch: 128
      ch_mult:
      - 1
      - 2
      - 4
      num_res_blocks: 2
      attn_resolutions: []
      dropout: 0.0
    lossconfig:
      target: taming.modules.losses.vqperceptual.VQLPIPSWithDiscriminator
      params:
        disc_conditional: false
        disc_in_channels: 3
        disc_start: 0
        disc_weight: 0.8
        codebook_weight: 1.0

then rename it as vq-4.yaml. it works when i use the vq-4 yaml file and ckpt in project 'taming-transformers'.
